backend/dal/Services/Admin/Concrete/AgencyService.cs

```diff
@@ -28,6 +31,54 @@ public class AgencyService : BaseService<Agency>, IAgencyService
         #endregion
 
         #region Methods
+        
+        /// <summary>
+        /// Get a page of agencies from the datasource.
+        /// </summary>
+        /// <param name="page"></param>
+        /// <param name="quantity"></param>
+        /// <returns></returns>
+        public Paged<Agency> Get(int page, int quantity)
+        {
+            return Get(new AgencyFilter(page, quantity));
+        }
+
+        /// <summary>
+        /// Get a page of agencies from the datasource.
+        /// The filter will allow queries to search for the following property values; Name, Description, ParentId.
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <returns></returns>
+        public Paged<Agency> Get(AgencyFilter filter = null)
+        {
+            this.User.ThrowIfNotAuthorized(Permissions.SystemAdmin);
```

Comment:
I think this makes sense for now, because this list is only used on the agency form.